packages/puppeteer-extra-plugin-block-resources/index.js

```diff
@@ -91,14 +94,50 @@ class Plugin extends PuppeteerExtraPlugin {
     return this.opts.blockedTypes
   }
 
+  /**
+   * Get the request interception resolution priority.
+   *
+   * Priority for Cooperative Intercept Mode can be configured either through `opts` or by modifying this property.
+   *
+   * @type {number} - A number for the request interception resolution priority.
+   */
+  get interceptResolutionPriority() {
+    return this.opts.interceptResolutionPriority
+  }
+
   /**
    * @private
    */
   onRequest(request) {
     const type = request.resourceType()
     const shouldBlock = this.blockedTypes.has(type)
-    this.debug('onRequest', { type, shouldBlock })
-    return shouldBlock ? request.abort() : request.continue()
+
+    // Requests are immediately handled if not using Cooperative Intercept Mode
+    const alreadyHandled = request.isInterceptResolutionHandled
+      ? request.isInterceptResolutionHandled()
+      : true
+
+    this.debug('onRequest', {
+      type,
+      shouldBlock,
+      alreadyHandled
+    })
+
+    if (alreadyHandled) return
+
+    if (shouldBlock) {
+      const abortArgs = request.continueRequestOverrides
```

Comment:
Are you using `continueRequestOverrides` purely for version detection? If so, it might make more sense to use  [abortErrorReason](https://github.com/puppeteer/puppeteer/blob/dc23b7535cb958c00d1eecfe85b4ee26e52e2e39/src/common/HTTPRequest.ts#L220) since this is an `abort()` call.

Reply:
I didn't know about this method, thanks! I changed it for the feature detection 👍🏻